test/utils/integration/integration.go

```diff
@@ -0,0 +1,233 @@
+package integration
```

Comment:
(nit) This file is used in the context of testing the `internal/controller/eventing/controller.go` functionality. I suggest to move it there to increase the cohesion of our code base and tests.

Reply:
I'd keep it as is, to avoid merge conflicts with [this](https://github.com/kyma-project/eventing-manager/pull/23) upcoming PR.